docs/iris/src/whatsnew/contributions_1.10/deprecate_2016-Feb-25_logging.txt

```diff
@@ -0,0 +1,5 @@
+* deprecated logging functions (currently used only for rules logging):  
+ :data:`iris.config.iris.config.RULE_LOG_DIR`,
+ :data:`iris.config.iris.config.RULE_LOG_IGNORE`,
+ :data:`iris.config.iris.config.IMPORT_LOGGER` and
```

Comment:
I see now that the line at the top of the iris ```__init__``` where it refers to this does actually *do* something :  I had just misread it.
Will revert